src/engine/loopingcontrol.cpp

```diff
@@ -138,6 +184,18 @@ LoopingControl::LoopingControl(QString group,
     m_pCOLoopMove = new ControlObject(ConfigKey(group, "loop_move"), false);
     connect(m_pCOLoopMove, SIGNAL(valueChanged(double)),
             this, SLOT(slotLoopMove(double)), Qt::DirectConnection);
+    m_pCOLoopMoveForward = new ControlPushButton(ConfigKey(group, "loop_move_forward"));
+    connect(m_pCOLoopMoveForward, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveForward(double)));
+    m_pCOLoopMoveBackward = new ControlPushButton(ConfigKey(group, "loop_move_backward"));
+    connect(m_pCOLoopMoveBackward, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveBackward(double)));
+    m_pCOLoopMoveForwardBeatloopSize = new ControlPushButton(ConfigKey(group, "loop_move_forward_beatloop_size"));
+    connect(m_pCOLoopMoveForwardBeatloopSize, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveForwardBeatloopSize(double)));
+    m_pCOLoopMoveBackwardBeatloopSize = new ControlPushButton(ConfigKey(group, "loop_move_backward_beatloop_size"));
+    connect(m_pCOLoopMoveBackwardBeatloopSize, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveBackwardBeatloopSize(double)));
```

Comment:
IMHO these four controls are not well designed to be part of the GUI or the mapping. 
I would either prefer to remove them form this PR or replace them with a usable solution. 
What are the usecases? How is this solved on CD players?